synapseclient/models/mixins/asynchronous_job.py

```diff
@@ -370,14 +378,14 @@ async def get_job_async(
         SynapseTimeoutError: If the job does not complete or progress within the timeout interval.
     """
     client = Synapse.get_client(synapse_client=synapse_client)
-    start_time = asyncio.get_event_loop().time()
+    start_time = time.time()
```

Comment:
This kind of time was causing the message to appear oddly. Moving to `time.time`, seems to have resolved it, as it seems to wait the expected time.

![image](https://github.com/user-attachments/assets/154863b2-cf83-4bbe-bcc6-34e6e2bf0be7)

Reply:
The default timeout was 60. The configuration to change the default wasn't exposed up through the functions that called it.